torch/_dynamo/variables/distributed.py

```diff
@@ -112,10 +117,20 @@ def call_method(
     ) -> "VariableTracker":
         from . import ConstantVariable
 
-        allowed_methods = ["__init__", "__setattr__"]
-        # placement types dynamo tracking allows only __init__
-        # and __setattr__ methods, the latter is for case like `Shard(dim)`
-        if name in allowed_methods:
+        # Placement types dynamo tracking only allows following methods
+        # and __setattr__  is for case like `Shard(dim)` and methods.
+        # Methods in the list must satisfy:
+        #    1. Input arguments are constants and do not need to be guarded on;
+        #    2. Output is constant with respect to their inputs
+        allowed_constant_methods = [
```

Comment:
Usually we name them as ```constant_fold_functions```.
https://github.com/pytorch/pytorch/blob/c4eab49ded3ebd3742d9d1380f1ef15a312c834d/torch/_dynamo/variables/torch.py#L129